basics/holder.ts

```diff
@@ -0,0 +1,39 @@
+import { Holder } from '@kiltprotocol/sdk-js'
+import { DidDocument, SignerInterface } from '@kiltprotocol/types'
+import { randomAsHex } from '@polkadot/util-crypto'
+
+import { Types } from '@kiltprotocol/credentials'
+
+export async function createCredentialPresentation(
+  credential: Types.VerifiableCredential[],
+  holderDid: DidDocument,
+  signers: SignerInterface[]
+): Promise<void> {
+  const challenge = randomAsHex()
+
+  const presentation = await Holder.createPresentation({
+    credentials: credential,
+    holder: { didDocument: holderDid, signers },
+    presentationOptions: {
+      validFrom: new Date(),
+      validUntil: new Date(Date.now() + 1000 * 60 * 60 * 24),
+      verifier: 'did:kilt:4s7XuQtwCfqtL2My9NKYXb6E8BvHqbdWooXocgwvhsYJjMgR',
+    },
+    proofOptions: { challenge },
+    now: new Date(),
```

Comment:
```suggestion
```
That is the default anyway, so not really necessary.

Reply:
Ok, I have updated this. However, I tried to add more of the proof options. It was a real mess. Not entirely sure. I could imagine doing multiple holder `createCredentialPresentation` in order to show case what people can do.